Source/DafnyCore/Backends/GoLang/GoCodeGenerator.cs

```diff
@@ -53,8 +60,10 @@ string FormatDefaultTypeParameterValue(TopLevelDecl tp) {
       return $"_default_{tp.GetCompileName(Options)}";
     }
 
+    private readonly Dictionary<ModuleDefinition, Import> ModuleImports = new();
```

Comment:
Do we still need `Imports` after we have this field?

Reply:
Renamed it to `ImportsNotFromDafnyModules` and updated the behavior accordingly